.github/workflows/gradle-pr-build.yml

```diff
@@ -35,10 +35,3 @@ jobs:
         run: ./gradlew build ktlintCheck
       - name: Build with dev Kotlin version
         run: ./gradlew -PkotlinDev build ktlintCheck
-      - name: Upload test results
-        uses: actions/upload-artifact@v3
-        if: failure()
-        with:
-          name: Test artifacts
-          path: '**/build/reports/tests/**'
-          if-no-files-found: warn
```

Comment:
I'm removing this action, you can view test results via Gradle build scan for now.